consensus/istanbul/backend/engine.go

```diff
@@ -544,8 +544,8 @@ func (sb *backend) Finalize(chain consensus.ChainReader, header *types.Header, s
 		}
 	}
 
-	// The Registry contract must be immediately available from the fork block.
-	// So it is installed at block (RandaoCompatibleBlock - 1) which is before the fork block.
+	// The Registry contract is installed at RandaoCompatibleBlock,
+	// so we need to call KIP113 contract from RandaoRegistry at RandaoCompatibleBlock.
```

Comment:
I'm not sure what the second line means..
How about just this?

```
	// The Registry and KIP113 contract are installed at RandaoCompatibleBlock
```

Reply:
LGTM, applied at https://github.com/klaytn/klaytn/pull/2111/commits/111feac9b749288f58da8586ced893eadea2d5b4